types/algorithm-metadata.go

```diff
@@ -7,4 +7,12 @@ type AlgorithmMetadata struct {
 	// Parameters is an array that contains all parameters that may be passed to
 	// the algorithm using the request body
 	Parameters map[string]Parameter `json:"parameters"`
+
+	// UseBuckets specifies if the data query uses buckets of time to
+	// pre-aggregate the data it receives
+	UseBuckets bool `json:"useBuckets" yaml:"useBuckets"`
+
+	// BucketSize specifies the size of each bucket in seconds.
+	// As the buckets may only be of a positive size, the values here are uint64
+	BucketSize string `json:"bucketSize" yaml:"bucketSize"`
```

Comment:
Why is this field a string?

Reply:
This field is a string, since the value read from the metadata files (e.g. `linear.yaml`) is string to cope with postgres intervals correctly

Reply:
Add this to the description of the type please.